app/mailers/user_mailer.rb

```diff
@@ -4,8 +4,10 @@ class UserMailer < ApplicationMailer
 
   def review_alert_email(user)
     @user = user
-    @url  = "http://www.chinatown-aces.herokuapp.com"
-    mail(to: @user.email, subject: "Information regarding your business")
+    @url  = 'http://www.chinatown-aces.herokuapp.com'
```

Comment:
Unnecessary spacing detected.<br>Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.